src/events/SDL_pen.c

```diff
@@ -269,7 +269,7 @@ static void pen_sort(void)
               pen_compare);
     pen_handler.sorted = SDL_TRUE;
 }
-
+#include<stdio.h>
```

Comment:
Was this intentional?

Reply:
This is probably a debug remainder.
I removed it in beb52ed54
Good catch :)